_episodes_rmd/06-best-practices-R.Rmd

```diff
@@ -3,11 +3,14 @@ title: "Best Practices for Writing R"
 teaching: 10
 exercises: 0
 questions:
-- "How can I write R that other people can understand and use?"
+- "What are the best practices for writing R so others can understand what I have done?"
```

Comment:
Since the title contains "best practices" already, I'd rather avoid a duplication.

Reply:
I wanted to change the question up a bit to make it easier to understand what the lesson is about. The objectives seemed to be a little more vague so I thought they needed to be updated a bit more than what was already there.

Reply:
Maybe change the question to "What are ways that I can write R so others can understand what I have done?"?

Reply:
I do think the revised statement is losing an idea that's captured in the original: it's not just having understandable code that's important but also that others are able to use/apply it. I don't think that idea is obvious to people new to programming.